Translations/Base_Italian.lang

```diff
@@ -1,35 +1,35 @@
-#GUI_PAGEABLE_SEARCH_TITLE: Search
-#GUI_PAGEABLE_NEXT_TITLE: Next
-#GUI_PAGEABLE_NEXT_ACTION: Click to go to the next page
-#GUI_BACK_ACTION: Click to go back
-#GUI_SETTINGS_MYSQL_SETUP_INFO: This requires an external MySQL Database to connect to
-#GUI_SETTINGS_MYSQL_RESTARTREQUIRED: You will be asked to restart to apply the changes
-#GUI_SETTINGS_LANGUAGE_NAME: Language
-#COMMAND_NOCONSOLE: This command cannot be run through the console
-#COMMAND_NOPERMISSION: You don't have **access** to that command
-#GUI_PAGEABLE_SEARCH_TYPEIN: Type in a search term
-#SETTINGS: Settings
-#ENABLE: enable
-#NONE: None
-#ENABLED: enabled
-#GUI_PAGEABLE_PREVIOUS_TITLE: Previous
-#GUI_SETTINGS_MYSQL_CURRENTVALUE: Current Value:
-#GUI_PAGEABLE_SEARCH_ACTION: Click to search
-#GUI_SETTINGS_SELECTPAGE: Click to show Settings
-#GUI_SETTINGS_PAGESELECTED: This page is currently **selected**
-#GUI_SETTINGS_MYSQL_CANTCONNECT: Could not connect to your MySQL
-#GUI_SETTINGS_MYSQL_SAVE_TITLE: Test & Save
+#GUI_SETTINGS_MYSQL_EDITVALUE_ACTION: CLicca per modificare questo valore
+#DISABLE: disattiva
+#GUI_SETTINGS_MYSQL_DISABLE_TITLE: Disattivare MySQL
+#DISABLED: disattivato
+#GUI_SETTINGS_MYSQL_DISABLE_ACTION: Clicca per disattivare MySQL
+#GUI_BACK_NAME: Indietro
+#GUI_PAGEABLE_SEARCH_CLOSE_ACTION: Clicca per **close** la ricerca
+#GUI_PAGEABLE_SEARCH_TITLE: Ricerca
+#GUI_PAGEABLE_NEXT_TITLE: Successivo
+#GUI_PAGEABLE_NEXT_ACTION: Clicca per andare alla pagina successiva
+#GUI_BACK_ACTION: Clicca per tornare indietro
+#GUI_SETTINGS_MYSQL_SETUP_INFO: Questo richiede un database esterno MySQL al quale connettersi
+#GUI_SETTINGS_MYSQL_RESTARTREQUIRED: Dovrai riavviare per applicare le modifiche
+#GUI_SETTINGS_LANGUAGE_NAME: Lingua
+#COMMAND_NOCONSOLE: Questo comando non può essere avviato dalla **console**
+#COMMAND_NOPERMISSION: Non hai **access** a quel comando
+#GUI_PAGEABLE_SEARCH_TYPEIN: Digitare il termine da cercare
+#SETTINGS: Impostazioni
+#ENABLE: abilita
+#NONE: niente
+#ENABLED: abilitato
+#GUI_PAGEABLE_PREVIOUS_TITLE: Precedente
+#GUI_SETTINGS_MYSQL_CURRENTVALUE: Valore attuale:
+#GUI_PAGEABLE_SEARCH_ACTION: Clicca per cercare
+#GUI_SETTINGS_SELECTPAGE: Clicca per mostrare le impostazioni
+#GUI_SETTINGS_PAGESELECTED: Questa pagina è attualmente **selected**
+#GUI_SETTINGS_MYSQL_CANTCONNECT: Impossibile connettersi al tuo MySQL
+#GUI_SETTINGS_MYSQL_SAVE_TITLE: Testa & Salva
 #GUI_SETTINGS_MYSQL_SETUP_TITLE_FIELDNAME: MySQL - %1
-#GUI_PAGEABLE_PREVIOUS_ACTION: Click to go to the previous page
-#GUI_SETTINGS_MYSQL_SAVE_ACTION: Click to **test** and **save**
-#GUI_SETTINGS_MYSQL_NAME: MySQL Database
-#GUI_SETTINGS_MYSQL_SETUP_ACTION: Click to enable MySQL
-#GUI_SETTINGS_MYSQL_SETUP_TITLE_TYPEINVALUE: Type in the value
-#GUI_SETTINGS_MYSQL_SETUP_TITLE: Setup MySQL
+#GUI_PAGEABLE_PREVIOUS_ACTION: Clicca per andare alla pagina precedente
+#GUI_SETTINGS_MYSQL_SAVE_ACTION: Clicca per **test** e **save**
+#GUI_SETTINGS_MYSQL_NAME: MySQL **Database**
```

Comment:
Please keep the format of the original phrases. That means no addition of ** in your language. Please remove them